tools/linter/adapters/s3_init_config.json

```diff
@@ -1,4 +1,10 @@
 {
+    "HOW TO UPDATE THE BINARIES": [
```

Comment:
So nothing is doing something for each entry in the dictionary? Seems hacky, but I guess OK.

Reply:
Yeah, nothing traverses the entire dictionary. `s3_init.py` (via `.lintrunner.toml`) is setup to just look up the value of the specific key they need.

It's a bit hacky, yeah, but the alternatives seemed worse